contracts/rewards/interfaces/IAutoCompoundingRewards.sol

```diff
@@ -93,6 +98,20 @@ interface IAutoCompoundingRewards is IUpgradeable {
      */
     function enableProgram(Token pool, bool status) external;
 
+    /**
+     * @dev sets the number of programs to process the rewards for, when auto-trigerred
+     *
+     * requirements:
+     *
+     * - the caller must be the admin of the contract
+     */
+    function setAutoTriggerCount(uint256 autoTriggerCount) external;
+
+    /**
+     * @dev processes program rewards based on internal logic, without requiring any input
+     */
+    function trigger() external;
```

Comment:
I'm not in love with the generic `trigger` name. Maybe something like `autoProcessRewards` or  `triggerProcessRewards`?

Reply:
`autoProcess`?

Reply:
`autoProcess` works too

Reply:
done (pretty massive rename)